addons/hr_timesheet/migrations/9.0.1.0/tests/test_hr_timesheet.py

```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
+from openerp.tests.common import TransactionCase
+
+
+class TestHrTimesheet(TransactionCase):
+    def test_hr_timesheet(self):
+        self.env.cr.execute(
+            "SELECT line_id FROM hr_analytic_timesheet LIMIT 1")
+        line_id, = self.env.cr.fetchone()
+        line = self.env['account.analytic.line'].browse(line_id)
+        self.assertTrue(line.is_timesheet)
```

Comment:
.. I believe, which means it has to be proposed and merged separately and then the database has to be created manually and uploaded. After that, the test can be run.

I feel this method still needs some work to be usable....